README.md

```diff
@@ -315,7 +315,28 @@ time.sleep(10)
 order_book.close()
 ```
 
+### Error Handling
+Error handling has been added in version 2.0. Currently, the only HTTP error codes that
+are handled are the ones documented on the GDAX API error section: [here](https://docs.gdax.com/?python#errors)
+
+- HTTP STATUS CODE 400: Raises InvalidGdaxRequest
+- HTTP STATUS CODE 401: Raises UnauthorizedGdaxRequest
+- HTTP STATUS CODE 403: Raises ForbiddenGdaxRequest
+- HTTP STATUS CODE 404: Raises NotFoundGdaxRequest
+- HTTP STATUS CODE 429: Raises GdaxRateLimitRequest
+- HTTP STATUS CODE 4XX: Raises UnknownGdaxClientRequest
+- HTTP STATUS CODE 5XX: Raises InternalErrorGdaxRequest
+
+All HTTP requests from both the public client and authenticated client run through `_determine_response`, which
+either returns the JSON body or raises the appropriate error.
+
+Please consider creating new Exception classes and mapping as Gdax error states are discovered.
+
+
 ## Change Log
+*2.0*
```

Comment:
Why are you bumping the version to 2.0, is it not backwards compatible anymore?

Reply:
This is not backwards compatible, as it will raise exceptions instead of returning a response object.